src/app/view/components/dot-base-type-selector/dot-base-type-selector.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { DotContentletService } from '../../../api/services/dot-contentlet.service';
+import { StructureTypeView } from '../../../shared/models/contentlet/structure-type-view.model';
+import { Observable } from 'rxjs/Observable';
+import { DotMessageService } from '../../../api/services/dot-messages-service';
+import { SelectItem } from 'primeng/primeng';
+import { map, take } from 'rxjs/operators';
+
+@Component({
+    selector: 'dot-base-type-selector',
+    templateUrl: './dot-base-type-selector.component.html',
+    styleUrls: ['./dot-base-type-selector.component.scss']
+})
+export class DotBaseTypeSelectorComponent implements OnInit {
+    @Input() value: SelectItem;
+    @Output() selected = new EventEmitter<string>();
```

Comment:
change is a better name

Reply:
we have being using `selected` in other components like persona o device selector to emulate the "natural" behavior of a select in html.